packages/fxa-settings/src/components/PageDeleteAccount/index.test.tsx

```diff
@@ -80,6 +80,26 @@ describe('PageDeleteAccount', () => {
     expect(screen.getByTestId('continue-button')).toBeDisabled();
   });
 
+  it('Requires at least 8 characters to enable Delete button', async () => {
+    renderWithRouter(
+      <AppContext.Provider value={mockAppContext({ account })}>
+        <PageDeleteAccount />
+      </AppContext.Provider>
+    );
+
+    await advanceStep();
+    expect(screen.getByTestId('delete-account-button')).toBeDisabled();
+
+    await act(async () => {
+      await typeByTestIdFn('delete-account-confirm-input-field')('hunter67');
+      expect(
+        screen.getByTestId('delete-account-confirm-input-field')['value'].length
+      ).toBeGreaterThanOrEqual(8);
+    });
+
+    expect(screen.getByTestId('delete-account-button')).toBeEnabled();
+  });
+
   it('Gets valid response on submit', async () => {
```

Comment:
@jodyheavener I did refactor the tests as you suggested but I'm still getting a test failure here at this test

Reply:
@Yogita98 Hmm, I'm not seeing the changes. Did you push them up? This test appears to be unchanged

Reply:
@Yogita98 Sorry, yes I see your changes to your new test, but you'll need to change the "Gets valid response on submit" test as well to use 8 characters instead of 7.

Reply:
Alright got it!

Reply:
@jodyheavener Let me know how the change looks like now! I'll probably have to squash my commits